From: =?utf-8?q?Jonas_Dre=C3=9Fler?= <verdre@v0yd.nl>
Date: Thu, 11 Feb 2021 17:30:17 +0100
Subject: clutter: Use event source directly for debug message

No need to assign things to a separate variable there, let's just read
event->any.source directly instead.

Part-of: <https://gitlab.gnome.org/GNOME/mutter/-/merge_requests/1729>
---
 clutter/clutter/clutter-main.c | 32 ++++++++++----------------------
 1 file changed, 10 insertions(+), 22 deletions(-)

diff --git a/clutter/clutter/clutter-main.c b/clutter/clutter/clutter-main.c
index ce2d2de..8e18e62 100644
--- a/clutter/clutter/clutter-main.c
+++ b/clutter/clutter/clutter-main.c
@@ -1857,7 +1857,6 @@ _clutter_process_event_details (ClutterActor        *stage,
       case CLUTTER_TOUCHPAD_PINCH:
       case CLUTTER_TOUCHPAD_SWIPE:
         {
-          ClutterActor *actor;
           gfloat x, y;
 
           clutter_event_get_coords (event, &x, &y);
@@ -1901,22 +1900,17 @@ _clutter_process_event_details (ClutterActor        *stage,
                   break;
                 }
 
-              actor = update_device_for_event (CLUTTER_STAGE (stage), event, TRUE);
-              if (actor == NULL)
-                break;
+              event->any.source =
+                update_device_for_event (CLUTTER_STAGE (stage), event, TRUE);
 
-              event->any.source = actor;
-            }
-          else
-            {
-              /* use the source already set in the synthetic event */
-              actor = event->any.source;
+              if (event->any.source == NULL)
+                break;
             }
 
           CLUTTER_NOTE (EVENT,
                         "Reactive event received at %.2f, %.2f - actor: %p",
                         x, y,
-                        actor);
+                        event->any.source);
 
           /* button presses and releases need a click count */
           if (event->type == CLUTTER_BUTTON_PRESS ||
@@ -1970,7 +1964,6 @@ _clutter_process_event_details (ClutterActor        *stage,
       case CLUTTER_TOUCH_CANCEL:
       case CLUTTER_TOUCH_END:
         {
-          ClutterActor *actor;
           gfloat x, y;
 
           clutter_event_get_coords (event, &x, &y);
@@ -2002,22 +1995,17 @@ _clutter_process_event_details (ClutterActor        *stage,
                   break;
                 }
 
-              actor = update_device_for_event (CLUTTER_STAGE (stage), event, TRUE);
-              if (actor == NULL)
-                break;
+              event->any.source =
+                update_device_for_event (CLUTTER_STAGE (stage), event, TRUE);
 
-              event->any.source = actor;
-            }
-          else
-            {
-              /* use the source already set in the synthetic event */
-              actor = event->any.source;
+              if (event->any.source == NULL)
+                break;
             }
 
           CLUTTER_NOTE (EVENT,
                         "Reactive event received at %.2f, %.2f - actor: %p",
                         x, y,
-                        actor);
+                        event->any.source);
 
           emit_touch_event (event, device);
 
